server/test/services/export/PdfExporterTest.java

```diff
@@ -395,8 +394,26 @@ public void exportProgram_hasMainProgramInfo() throws IOException {
     assertThat(linesFromPdf.get(1)).isEqualTo("Admin name: " + programDef.adminName());
     assertThat(linesFromPdf.get(2))
         .isEqualTo("Admin description: " + programDef.adminDescription());
-    assertThat(linesFromPdf.get(3)).contains("Time of export:");
-    assertThat(linesFromPdf.get(4)).isEqualTo("Origin of export: http://localhost:9000");
+    assertThat(linesFromPdf.get(3))
+        .contains(
+            "Admin short description: " + programDef.localizedShortDescription().getDefault());
+    assertThat(linesFromPdf.get(4)).contains("Time of export:");
+    assertThat(linesFromPdf.get(5)).isEqualTo("Origin of export: http://localhost:9000");
+
```

Comment:
Nit: We can also assert that newlines are inserted on line 6 and 10.

Reply:
Done!